doc_src/interactive.rst

```diff
@@ -523,7 +523,7 @@ After a command has been executed, it is remembered in the history list. Any dup
 
 By pressing :kbd:`Alt`\ +\ :kbd:`↑` and :kbd:`Alt`\ +\ :kbd:`↓`, a history search is also performed, but instead of searching for a complete commandline, each commandline is broken into separate elements just like it would be before execution, and the history is searched for an element matching that under the cursor.
 
-History searches are case-insensitive unless the search string contains an uppercase character, and they can be aborted by pressing the escape key.
+History searches are case-insensitive unless the search string contains an uppercase character, and they can be aborted or edited by pressing :kbd:`Esc`.
```

Comment:
(Any movement key (like left-arrow) also stops history search. That should be fairly obvious from trial and error.)

I don't think "edited" is accurate here because you can only start a new search.

Reply:
ah, you're right, my bad! Page-Down and Control-Z also do that.

Reply:
Really? I don't think pgdn or ^Z are documented. Should they be?

Reply:
I have added my suggestion (because you don't edit the search but the *search string*).

I added page-down, but not the various keys for undo (ctrl-Z, ctrl-_, ctrl-/).
I guess undo is a general feature, and it even has a glitch here:
press `echo` <kbd>Up</kbd> ^Z <kbd>Up</kbd> ^Z
The first undo goes back to the original string but the second clears the command line.